### ISSUE TITLE
Get-RSCCluster -detail not working

### ISSUE THREAD

**Issue by doslagerRubrik** [rubrikinc/rubrik-powershell-sdk#40]
I tried to use the -detail parameter for Get-RSCCluster and got the following output. I just tried this param for the first time. Without the -detail param it works fine and returns basic details. 

PS 7.4.0 (MacOS)
Module version 0.24 installed via PowerShell Gallery

```
PS /> get-rscCluster -Name xxxxxxx -Detail
Invoke-Rsc: /Users/xxxxxxx/.local/share/powershell/Modules/RubrikSecurityCloud/0.24/Toolkit/Public/Get-RscCluster.ps1:163
Line |
 163 |          $response = Invoke-Rsc $query
     |                      ~~~~~~~~~~~~~~~~~
     |  One or more errors occurred. (Unable to cast object of type 'Newtonsoft.Json.Linq.JValue' to type 'Newtonsoft.Json.Linq.JObject'.) Unable to cast object of type
     | 'Newtonsoft.Json.Linq.JValue' to type 'Newtonsoft.Json.Linq.JObject'.
PS />
```

**Comment by doslagerRubrik** [comment]
Also, same result on Windows PS 7.4.1

**Comment by guirava** [comment]
Fixed in 0.30
